### PR TITLE
Add option for favorites list and quick add button (MOOS-31)

### PR DESCRIPTION
**Pull request by tulimaki**
- Add option to show favorites wishlist at dashboard menu
- Add option to quick add products to favorites wishlist
- Tune texts so that we are not expecting that it is products we add to lists
- Also there is now option to only show favorites option as a lightweight wishlist

### CODE REVIEW

**Comment by tulimaki** [comment]
Note: Once this is cool I double check those license headers.

File: shuup_wishlist/configuration.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+def is_favorites_enabled(shop):
+    return configuration.get(shop, "show_favorites_wishlist_on_dashboard", False)
+
+
+class WishlistSettingsForm(BaseSettingsForm):
+    title = _("Wishlist Settings")
+    show_wishlist_on_dashboard = forms.BooleanField(
+        label=_("Show wishlists on dashboard"),
+        help_text=_("Setting this to false will hide wishlisth from customer dashboard."),
```
Comment (chessbr) [inline shuup_wishlist/configuration.py]: typo

File: shuup_wishlist/dashboard_items.py [hunk 2]
```diff
@@ -19,10 +23,33 @@ class WishlistItem(DashboardItem):
     view_text = _("Show all")
     _url = "shuup:personal_wishlists"
 
+    def show_on_menu(self):
+        return is_dasbhoard_enabled(self.request.shop)
+
+    def show_on_dashboard(self):
+        return is_dasbhoard_enabled(self.request.shop)
```
Comment (chessbr) [inline shuup_wishlist/dashboard_items.py]: this returns the same thing as the previous function

File: shuup_wishlist/favorites.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+# Copyright (c) 2012-2018, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from shuup_wishlist.models import Wishlist
+
+
+def get_favorites_list(shop, customer):
+    return Wishlist.objects.get_or_create(
+        shop=shop, customer=customer, name="Favorites")[0]
```
Comment (chessbr) [inline shuup_wishlist/favorites.py]: translate the name?

Comment (tulimaki) [reply shuup_wishlist/favorites.py]: I guess

File: shuup_wishlist/dashboard_items.py (hunk 2 above)
Comment (tulimaki) [reply shuup_wishlist/dashboard_items.py]: This is ok

**Pull request merged**